docs/content/2.deploy/20.providers/zeabur.md

```diff
@@ -0,0 +1,18 @@
+# Zeabur
+
+Deploy Nitro apps to [Zeabur](https://zeabur.com).
+
+**Preset:** `zeabur` ([switch to this preset](/deploy/#changing-the-deployment-preset))
+
+::alert
+**Zero Config Provider**
+:br
+Integration with this provider is possible with zero configuration. ([Learn More](/deploy/#zero-config-providers))
```

Comment:
This is amazing! But to be a zero-config provider, I expect the platform's CI/CD to expose a known environment variable which we can detect with [std-env](https://github.com/unjs/std-env) and also nitro to auto switch the preset.

Reply:
Thanks! Feel free to open a PR for std-env also once ready 👍🏼